### ISSUE TITLE
toggle title includes focusable anchor text inside of a div with role of button--accessibility issue

### ISSUE THREAD

**Issue by Trott** [elementor/elementor#22977]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

The latest Elementor uses `<div ... role="button" ... >` for the clickable toggle title. Cool. That's legit.

Unfortunately, inside that div, there is `<a ... tabIndex="0" ... >`. Accessibility tools (correctly, I believe) call that out as an issue as a button should be focusable, but things inside the button should not be focusable.

I am floating a patch on top of Elementor to remove the hardcoded `tabIndex="0"` and it seems to both fix the accessibility issue (as far as the accessibility tools are concerned, at least) and does not (as far as I can tell) negatively impact the widget functionality.

### Steps to reproduce

1) Create a toggle.
2) Run an accessibility checker like SiteImprove.
3) See results identifying something like https://help.siteimprove.com/support/solutions/articles/80001058756-accessibility-rule-role-with-implied-hidden-content-has-keyboard-focus-explained as the problem.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

					
== Server Environment ==
	Operating System: Linux
	Software: nginx/1.21.6
	MySQL version: MariaDB Server v10.4.25
	PHP Version: 7.4.30
	PHP Memory Limit: 512M
	PHP Max Input Vars: 10000
	PHP Max Post Size: 100M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: There are some writing permissions issues with the following directories/files:
		 - WordPress root directory
	Elementor Library: Connected

**Comment by Trott** [comment]
Suggested fix is in https://github.com/elementor/elementor/pull/22978, although you'll probably want to do the same thing for accordions.

**Comment by Yerokhov** [comment]
Hi! You are using invalid html semantics in this element. In your code, you are using the <a> tag instead of the native <details> See [this](https://www.w3schools.com/tags/tag_details.asp). Because of this, the Google Page Speed service gives errors because it cannot crawl links. Error text "The link cannot be scanned. To crawl websites, search engines can use "href" attributes in links. Make sure the "href" attribute of the anchor elements redirects to the right place so that more pages of the site can be crawled."

**Comment by nicholaszein** [comment]
Hi @Yerokhov,


The warning you seen in GPS is a **<ins>FALSE POSITIVE</ins>**. You should not worry about it. These links lead to nothing, should not be crawled anyway, and the content inside both Accordion and Toggle widgets get crawled nonetheless, regardless if the `<a>` tag is there or not. 

We are aware that the semantics of both these widgets is incorrect, but we will not change them without implementing a safe way to do it that doesn't break backward compatibility. If we change them, it would be part of an Experiment.

You can very soon use the new Nested Accordion widget that will be released with Elementor 3.15, and has semantic html on its markup by default.

Kind regards
